test/e2e/storage/persistent_volumes-local.go

```diff
@@ -159,12 +158,13 @@ var _ = utils.SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolum
 
 		ssTester := framework.NewStatefulSetTester(f.ClientSet)
 		config = &localTestConfig{
-			ns:       f.Namespace.Name,
-			client:   f.ClientSet,
-			nodes:    nodes.Items,
-			node0:    node0,
-			scName:   scName,
-			ssTester: ssTester,
+			ns:           f.Namespace.Name,
+			client:       f.ClientSet,
+			nodes:        nodes.Items,
+			node0:        node0,
+			scName:       scName,
+			ssTester:     ssTester,
+			discoveryDir: filepath.Join(hostBase, fmt.Sprintf("%v-%v", f.Namespace.Name, rand.Int())),
```

Comment:
They should not be overlapping because the it uses the test namspace, which is different for each test case.  I think actually the rand.Int() is unnecessary.

Reply:
done